tests/test_TranscriptTbxFile.py

```diff
@@ -0,0 +1,11 @@
+from efs_parser.TranscriptTbxFile import TranscriptTbxFile
+
+#ee = TranscriptTbxFile("transcripts.tsv.bgz")
+ee = TranscriptTbxFile("tests/transcripts.tsv.bgz")
+def test_columns():
+    assert (ee.columns==['chr', 'start', 'end', 'strand', 'transcript_id', 'exon_starts', 'exon_ends', 'gene'])
+
+def test_range():
+    gf= ee.getRange('chr1',11874,944574)
```

Comment:
and finally here - also here - make sure `res` only contain rows within the given range